cmd/platform/channel.go

```diff
@@ -110,6 +110,7 @@ func init() {
 
 	modifyChannelCmd.Flags().Bool("private", false, "Convert the channel to a private channel")
 	modifyChannelCmd.Flags().Bool("public", false, "Convert the channel to a public channel")
+	modifyChannelCmd.Flags().String("username", "", "Required. Username who changes the channel privacy.")
```

Comment:
@grundleborg Do we want to make this a required field? Are we going to make it optional once we have something like Matterbot implemented?

Reply:
This is has to be required for now, but can become optional once we have a better long-term solution such as matterbot.

Reply:
@grundleborg I also assume this could be a breaking change for existing installs that use the command without specifying the username, then?

Reply:
Yes I suppose so, if someone has automated running that CLI command for some reason. I'd just note it in the release notes rather than worrying to much (0/5).